src/methods/get.js

```diff
@@ -344,7 +344,10 @@ export function OrdinaryGetPartial(
   Receiver: Value
 ): Value {
   if (Receiver instanceof AbstractValue && Receiver.getType() === StringValue && P === "length") {
-    return AbstractValue.createFromTemplate(realm, lengthTemplate, NumberValue, [Receiver], lengthTemplateSrc);
+    return AbstractValue.createTemporalFromTemplate(realm, lengthTemplate, NumberValue, [Receiver], {
```

Comment:
This is a simple example. Given that the type of the Receiver is a string, this can never throw or have side-effects. It can only error if this is fed with the wrong type of input arg. That needs to be fixed.

However, once we fix that, how will I know that this can be turned atemporal again? Not everything has this property. So I have to go through them all one-by-one and think hard if this can really throw or not.

Reply:
Simple: if Receiver is temporal, the get call must be temporal. Otherwise it is a-temporal.